From: Andrew Morton <akpm@linux-foundation.org>
Subject: arm-mm-drop-vm_fault_badmap-vm_fault_badaccess-checkpatch-fixes
Date: Thu Apr 11 01:17:26 PM PDT 2024

ERROR: code indent should use tabs where possible
#86: FILE: arch/arm/mm/fault.c:368:
+        code = SEGV_MAPERR;$

WARNING: please, no spaces at the start of a line
#86: FILE: arch/arm/mm/fault.c:368:
+        code = SEGV_MAPERR;$

total: 1 errors, 1 warnings, 68 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/arm-mm-drop-vm_fault_badmap-vm_fault_badaccess.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/arm/mm/fault.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/mm/fault.c~arm-mm-drop-vm_fault_badmap-vm_fault_badaccess-checkpatch-fixes
+++ a/arch/arm/mm/fault.c
@@ -365,7 +365,7 @@ done:
 	if (likely(!(fault & VM_FAULT_ERROR)))
 		return 0;
 
-        code = SEGV_MAPERR;
+	code = SEGV_MAPERR;
 bad_area:
 	/*
 	 * If we are in kernel mode at this point, we
_
